### ISSUE TITLE
Deleting Accounts/Journals

### ISSUE THREAD

**Issue by ghost** [PrudentMe/main#15]
Hi, 

How do we delete accounts/journals?

**Comment by vitoc** [comment]
Accounts are derived from transactions. Once there are no transactions with an account, the account will stop appearing in the account tree and etc.

Journals are just text files that you can delete from your file explorer. 

Hope this helps!

**Comment by ghost** [comment]
Thank you Vito.

One more question, how do we create all the different months? I’ve been stuck on November’s.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by vitoc** [comment]
You don't need to create any month, try clicking on the left or right button beside the Today button to move around months. 
![Today](https://user-images.githubusercontent.com/558813/201477740-caadf0e9-4801-41bd-acd1-721aa896e31f.png)

**Comment by ghost** [comment]
Thank you.

Any plans for DBS pdf statements - for credit cards or more than 6 months transactions?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ghost** [comment]
Hi sorry,

Ignore the credit card question.
If I add multiple journals, how do I ensure I’m just seeing the record of
one journal? It seems now the numbers are reflecting sum of consolidated
journals.

Thank you.

**Comment by vitoc** [comment]
Yes, the numbers in the account tree and reports are consolidated from all transactions across all open journals. To consider only one journal, please close the other journals.
